sdcm/nemesis.py

```diff
@@ -3734,6 +3742,24 @@ def disrupt_add_remove_dc(self) -> None:
             if node_added:
                 self.cluster.decommission(new_node, soft_timeout=MAX_TIME_WAIT_FOR_DECOMMISSION)
 
+    def disrupt_sla_increase_shares_during_load(self):
+        prometheus_stats = PrometheusDBStats(host=self.monitoring_set.nodes[0].external_address)
```

Comment:
Don't we need here a check if it's enterprise and "raise unsupported nemesis" if it doesn't?

Reply:
Added